charts/images.yaml

```diff
@@ -44,7 +44,7 @@ images:
 - name: csi-driver-disk
   sourceRepository: github.com/kubernetes-sigs/azuredisk-csi-driver
   repository: mcr.microsoft.com/k8s/csi/azuredisk-csi
-  tag: "v1.1.1"
+  tag: "v1.2.0"
```

Comment:
Do you want drop support for k8s <= 1.16?
I might have wrong understanding of the [this](https://github.com/kubernetes-sigs/azuredisk-csi-driver#container-images--kubernetes-compatibility).
On the other side, there is versions 1.3.0 and 1.4.0.
Could you add short explanation why you have chosen to update to version 1.2.0?

Reply:
> Do you want drop support for k8s <= 1.16?
> I might have wrong understanding of the [this](https://github.com/kubernetes-sigs/azuredisk-csi-driver#container-images--kubernetes-compatibility).

On azure, azuredisk-csi or in general CSI is enabled only for v1.21 clusters. So these version constraints don't affect us currently.

> On the other side, there is versions 1.3.0 and 1.4.0.
> Could you add short explanation why you have chosen to update to version 1.2.0?

Initially user reported that he/she tried to use `Premium_ZRS` disks on Kubernetes v1.21. The operation failed with reason: 

```
  "error": {
    "code": "InvalidParameter",
    "message": "Storage account type 'Premium_ZRS' is supported by Microsoft.Compute API version 2020-12-01 and above.",
    "target": "dataDisk.managedDisk.storageAccountType"
  }
```

Then @MSSedusch and @kon-angelo found that support for ZRS disks is added in azuredisk-csi@v1.2.0 - see https://github.com/kubernetes-sigs/azuredisk-csi-driver/releases/tag/v1.2.0. Of course, I known that there are even newer versions but currently my focus was first to update to a version with ZRS support to tackle the issue I mentioned.

Reply:
Thanks for the info!